dags/dag.py

```diff
@@ -26,11 +26,26 @@ def login_test(**context):
 def login_test_twee(**context):
     try:
         import requests
-        page = requests.get(url)
+        from urllib3.exceptions import InsecureRequestWarning
```

Comment:
Voor nu ok. Als dit het oplost, moeten we daar wel wat mee voordat we naar echte productie gaan

Reply:
idd!